plex/Home/GUIWindowHome.cpp

```diff
@@ -754,6 +757,20 @@ void CGUIWindowHome::UpdateSections()
   bool havePlaylists = false;
   bool havePlayqueues = false;
 
+  if (g_advancedSettings.m_bSharedSectionsOnHome && g_plexApplication.dataLoader->HasSharedSections())
```

Comment:
Instead of repeating this code for shared and owned stuff wouldn't it make sense to just add shared sections to the section list if that advanced setting is enabled?